framework/src/fvbcs/FVFluxBC.C

```diff
@@ -75,7 +75,44 @@ template <ComputeStage compute_stage>
 void
 FVFluxBC<compute_stage>::computeJacobian(const FaceInfo & fi)
 {
-  // Jacobians are not evaluated right now.
+  _face_info = &fi;
+  _normal = fi.normal();
+  DualReal r = fi.faceArea() * computeQpResidual();
+
+  auto & sys = _subproblem.systemBaseNonlinear();
+  unsigned int dofs_per_elem = sys.getMaxVarNDofsPerElem();
+  unsigned int var_num = _var.number();
+  unsigned int nvars = sys.system().n_vars();
+
+  // We might currently be running on a face for which this kernel's variable
+  // is only defined on one side. If this is the case, we need to only
+  // calculate+add the residual contribution if there is a dirichlet bc for
+  // the active face+variable.
+  //
+  // TODO: Running this query here is slow and redundant
+  // - since we run the same query inside the MooseVariableDataFV when
+  // computing the ghost cell values.  Perhaps we can mark a boolean on the
+  // FaceInfo object when we run the query there and just check it on the
+  // FaceInfo object here.
+  std::vector<FVDirichletBC *> bcs;
+  _subproblem.getMooseApp()
+      .theWarehouse()
+      .query()
+      .template condition<AttribSystem>("FVBoundaryCondition")
+      .template condition<AttribThread>(_tid)
+      .template condition<AttribBoundaries>(fi.boundaryIDs())
+      .template condition<AttribVar>(_var.number())
+      .template condition<AttribInterfaces>(Interfaces::FVDirichletBC)
+      .queryInto(bcs);
+  bool have_dirichlet_bcs = bcs.size() > 0;
```

Comment:
You copied this from the fv kernel class, but it isn't needed here.  delete it.

Reply:
I think I did delete this at some point. Maybe later.

Reply:
You're right